examples/cpp/power/Makefile

```diff
@@ -16,7 +16,7 @@ power.wasm: gen
 	    power.c
 
 .PHONY: test
-test: debug
+test: power.wasm
```

Comment:
Changing this dependency to `power.wasm` implies that the test will always be built in *release* mode because the `DBGFLAGS` variable won't be set.  Perhaps we should set `DBGFLAGS` globally and then have the `release:` target clear it?